howso/ablation.amlg

```diff
@@ -139,6 +139,77 @@
 		(call !CreateCaseWeights (assoc feature_name weight_feature) )
 	)
 
+	;reduce a model by removing cases which have an influence weight entropy that falls above
+	; a threshold
+	;
+	;parameters:
+	; context_features: a list of context features to use when computing influence weight entropies
+	; distribute_weight_feature: optional, default '.case_weight'. name of feature whose values to use as case weights
+	; influence_weight_entropy_threshold: optional, default 0.6. quartile to use as the cutoff threshold for
+	;influence weight entropy when deciding which cases to remove.
```

Comment:
add a tab or two between the ; and the words to denote that this is a continuation of the previous line and not a new parameter.